src/OpenTelemetry/Metrics/MeterProviderSdk.cs

```diff
@@ -125,6 +125,15 @@ internal sealed class MeterProviderSdk : MeterProvider
                         var metricStreamConfig = viewConfig(instrument);
                         if (metricStreamConfig != null)
                         {
+                            if (metricStreamConfig is HistogramConfiguration histogramConfiguration)
+                            {
+                                // Validate the histogram bounds
+                                if (!this.IsSortedAndDistinct(histogramConfiguration.BucketBounds))
```

Comment:
Is there a way I can detect that this value is not the default?
In that case the validation check could be omitted.